src/main/kotlin/no/nav/helse/fritakagp/processing/gravid/soeknad/GravidSoeknadPDFGenerator.kt

```diff
@@ -70,9 +70,12 @@ class GravidSoeknadPDFGenerator {
     }
 
     private fun PDPageContentStream.writeTextWrapped(text: String, spacing: Int = 1) {
-        WordUtils.wrap(text.replace("\t", " "), 100).split('\n').forEach {
-            this.newLineAtOffset(0F, -LINE_HEIGHT * spacing)
-            this.showText(it)
-        }
```

Comment:
> Jeg ser ikke hvorfor denne er avhengig av OS. Vanligvis pleier det å være ulik håndtering av linjeskifte med CRLF (windows) eller LF (unix). Kan du utdype noe?

@laukvik Svarer her, for det blir lettere med kommentartråd.

Funksjonen `WordUtils.wrap("string", 100)` bruker OS-spesifikk linjeseparator. Dvs. at den inserter `\r\n` i input-strengen på Windows og `\n` på Mac. I neste del av pipen så bruker man `\n`, så det er kombinasjonen av dem som skaper krøll på min PC.

Ved å endre `WordUtils.wrap("string", 100)` til `WordUtils.wrap("string", 100, "\n", false)`, så fikser vi problemet ved å spesifisere hvilket linjeseparator som skal brukes. Det siste argumentet må med fordi `WordUtils.wrap` bare finnes i versjoner med to eller fire parametre.